src/server/sendLayoutHTTPResponse.js

```diff
@@ -129,7 +129,7 @@ function serializeChildNodes(args) {
 
     if (!inRouterElement && isApplicationNode(node)) {
       serialize = serializeApplication;
-    } else if (isRouteNode(node)) {
+    } else if (!inRouterElement && isRouteNode(node)) {
```

Comment:
Similar to line 130 - we render `<route>` elements differently if it's inside of a `<single-spa-router>` element versus when it isn't. The `<route>` element shows up outside of `<single-spa-router>` elements when we're rendering the server content, and it shows up inside when we're sending the layout definition to the browser (so that the browser can pick up where the server left off.